### PR TITLE
Insilico prefs sizing fix

### PR DESCRIPTION
**Pull request by mstucky**
PT #174869184:  Prefs panel location for embedded NGCHM.  

Notes:  The original code and Chris' changes have been incorporated. In addition, the original fix only solved the top/bottom sizing and placement of the prefs panel. The left/right placement was set to 0 right and 0 left, respectively. This had the panels being drawn off the NGCHM widget on embedded maps. Now the left right placement has also been modified to place the prefs panel inside the NGCHM widget.

### CODE REVIEW

**Review by mryaninsilico — APPROVED** [review]

**Review by marohrdanz — COMMENTED** [review]
I'm not actually sure about how to test this. I know it was an issue for other applications, but I don't know how those applications were using the NGCHM code. 

I tried just putting the viewer in an iframe on another page, and the prefs panel looks fine. But the prefs panel also looks fine when I do the same thing with the code in master. So I must be doing something wrong.

**Comment by mstucky** [comment]
Hi Mary,   I believe that Chris and Jun are going to test this as they want to test it in GenePattern Notebook. Chris is on here as an approver so we can leave it to him.

**Review by ChrisWakefield — APPROVED** [review]

**Pull request merged**
